pages/desktop/consumer_pages/base.py

```diff
@@ -84,6 +86,29 @@ class HeaderRegion(Page):
         def is_user_logged_in(self):
             return self.is_element_visible(*self._settings_toggle_locator)
 
+        @property
+        def categories_name(self):
+            return self.selenium.find_element(*self._categories_toggle_locator).text
+
+        def open_categories_menu(self):
+            categories = self.selenium.find_element(By.ID, 'header--categories')
+            self.selenium.find_element(*self._categories_toggle_locator).click()
+            WebDriverWait(self.selenium, self.timeout).until(self.element_settled(categories))
+
+        class element_settled(object):
```

Comment:
It would be most appropriate to put it in an `expected.py`. If you like it, I can create that now and a next step might be to extract it out to a shared library so other projects can use it. At least we can start a convention now.